lib/logux/error_renderer.rb

```diff
@@ -16,9 +17,14 @@ def message
         build_message(exception, exception.message)
       when StandardError
         # some runtime error that should be fixed
-        ['error', 'Please look server logs for more information']
+        if Logux.configuration.render_backtrace_on_error
```

Comment:
Hey! What do you think about extracting this code to new private method:
```ruby
def render_stardard_error(exception)
  if Logux.configuration.render_backtrace_on_error
    ['error', exception.backtrace]
  else
    ['error', 'Please look server logs for more information']
  end
end
```
to avoid rubucop warning and make code a little bit clearly

Reply:
done